bin/plugin/commands/changelog.js

```diff
@@ -97,6 +99,21 @@ const LABEL_TYPE_MAPPING = {
 	'[Type] Security': 'Security',
 };
 
+const LABEL_FEATURE_MAPPING = {
+	'[Package] Block Editor': 'Block Editor',
+	'[Package] Editor': 'Editor',
+	'[Package] Edit Widgets': 'Widgets',
+	'[Package] Widgets Customizer': 'Widgets',
+	'[Feature] Widgets Screen': 'Widgets',
+	'[Block] Legacy Widget': 'Widgets',
+	'[Package] Components': 'Components',
+	'[Feature] UI Components': 'Components',
+	'[Feature] Component System': 'Components',
+	'[Package] Block Library': 'Block Library',
+	'[Feature] Blocks': 'Block Library',
```

Comment:
Might be good to get clarification on this. It might be more Block API or something like that than Block Library.

Reply:
Good point.